source/topics/account/how-to-manage-billing.md

```diff
@@ -0,0 +1,25 @@
+For billing information, navigate to your account settings in the Aptible dashboard and click on Billing. 
+
+1. Under the _Overview_ tab, you can see: 
+
+    - Your current plan and monthly base rate
+    - The number of resources (containers, disk space, and domains) deployed per environment  _(Note: At time of writing, the dashboard does not yet reflect custom [container sizes](https://github.com/aptible/dashboard.aptible.com/issues/516))_
```

Comment:
While linking to issues isn't usually a great idea, this one is so helpful in clarifying that I think it belongs.